app/components/AppNavigator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { createStackNavigator } from 'react-navigation';
+import Login from '../view/Login';
+import Signup from '../view/Signup';
+import Home from '../view/Home';
+import pokemonDetail from '../components/PokemonDetails';
```

Comment:
you should use a screen folder for all screens, its confuse the folder view and components , the components should be stateless and only return UI and the screen stateful with states.